tests/security/test_tls.py

```diff
@@ -25,11 +25,11 @@ def test_tls_disabled(host: str, protocol_name: str):
 
     with pytest.raises(ssl.SSLError):
         context = ssl.SSLContext(protocol)
-        with socket.create_connection((str(host), PORT), timeout=10) as sock:
-            with context.wrap_socket(sock, server_hostname=str(host)) as ssock:
-                ssock.do_handshake()
-                # If we reach this point, the protocol is enabled (test should fail)
-                pytest.fail(f"Protocol {protocol} should be disabled but connection succeeded")
+        with socket.create_connection((str(host), PORT), timeout=10) as sock, \
```

Comment:
What's the rationale? Are nested `with` considered bad practice?

Reply:
less indentation, likely?

Reply:
Ok, a really minor improvement then, and in this case, in my view, defeated by the need to go multiline. My personnal code scanning system (eyes + brains) catch successive short with a lot better than they read wrapped lines.